Automattic/jetpack
# Update ESLint config for TypeScript files

Updates ESLint config as discussed in https://github.com/Automattic/jetpack/pull/40555#issuecomment-2536305040

## Proposed changes:

* Disable certain JSDoc related ESLint rules for TypeScript files which make types redundant.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:

* Open a TS file
* Remove the JSDoc type for any param
* Remove the JSDoc type for return
* Confirm that there is no error
* Do the same for a js file
* Confirm that the errors show up as before
